action.yaml

```diff
@@ -172,10 +177,10 @@ runs:
         # Run 'trunk check' on push
         ${GITHUB_ACTION_PATH}/push.sh
       env:
-        GITHUB_EVENT_AFTER: ${{ github.event.after }}
-        GITHUB_EVENT_BEFORE: ${{ github.event.before }}
-        GITHUB_REF_NAME: ${{ github.ref_name }}
-        GITHUB_REPOSITORY: ${{ github.repository }}
+        GITHUB_EVENT_AFTER: ${{ env.GITHUB_EVENT_AFTER || github.event.after }}
```

Comment:
Will we be the ones overriding these? If so, should we name them to make that clear?

Reply:
We could, I guess. I don't have strong opinions about the naming here.